### DISCUSSION TITLE
Dead Link In "Welcome to GitHub Copilot Technical Preview!"  Email

### DISCUSSION THREAD

**Discussion by JasonHoku** [community/community#6640] [Projects and Issues] [answered]
In the last paragraph of the welcome email, the link labeled "discussion forum" leads to a 404

Also, omg I'm in! 😍

**Comment by davecheney** [comment, marked as answer]
Sorry about the dead link, we had a few teething problems with this mornings round of invites. We'll get it sorted out for the next round, you can find some more information in this discussion.

https://github.com/github/feedback/discussions/6632

**Reply by JasonHoku** [reply]
Hey no problem at all, is it weird that I was excited by it to file a bug report?

**Comment by Jobrien0394** [comment]
You can just add your github account to vscode and install the github copilot extension, it worked for me.
